#update manging-the-automatic-deletion-of-branches

terbiar opened the pull request:
<!--
Thank you for contributing to this project! You must fill out the information below before we can review this pull request. By explaining why you're making a change (or linking to a pull request) and what changes you've made, we can triage your pull request to the best possible team for review.

See our [CONTRIBUTING.md](/main/CONTRIBUTING.md) for information how to contribute.

For changes to content in [site policy](https://github.com/github/docs/tree/main/content/github/site-policy), see the [CONTRIBUTING guide in the site-policy repo](https://github.com/github/site-policy/blob/main/CONTRIBUTING.md).

We cannot accept changes to our translated content right now. See the [contributing.md](/main/CONTRIBUTING.md#earth_asia-translations) for more information.

Thanks again!
-->

### Why:

Closes [issue link]

<!-- 
- If there's an existing issue for your change, please link to it.
- If there's _not_ an existing issue, please open one first to make it more likely that this update will be accepted: https://github.com/github/docs/issues/new/choose. -->

### What's being changed:

<!-- Share artifacts of the changes, be they code snippets, GIFs or screenshots; whatever shares the most context. -->

### Check off the following:

- [ ] I have reviewed my changes in staging (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**).
- [ ] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/CONTRIBUTING.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

<!-- Description of the writer impact here -->

terbiar committed:
#update manging-the-automatic-deletion-of-branches

imranshari commented:
A50s

yikin1988 commented:
> ### Why:
> Closes [issue link]
> 
> ### What's being changed:
> ### Check off the following:
> * [ ]  I have reviewed my changes in staging (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**).
> * [ ]  For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/CONTRIBUTING.md#self-review).
> 
> ### Writer impact (This section is for GitHub staff members only):
> * [ ]  This pull request impacts the contribution experience
>   
>   * [ ]  I have added the 'writer impact' label
>   * [ ]  I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

terbiar commented:
Jap. Nama kau mcm . Mlm tadi aku ada tngok kau add dkat ml beta?  Eh, aku
dah agak. Lvl bru lg.


On Sun, 6 Jun 2021, 10:15 yikin1988, ***@***.***> wrote:

> Why:
>
> Closes [issue link]
> What's being changed: Check off the following:
>
>    - I have reviewed my changes in staging (look for the
>    *deploy-to-heroku* link in your pull request, then click *View
>    deployment*).
>    - For content changes, I have completed the self-review checklist
>    <https://github.com/github/docs/blob/main/CONTRIBUTING.md#self-review>.
>
> Writer impact (This section is for GitHub staff members only):
>
>    -
>
>    This pull request impacts the contribution experience
>    - I have added the 'writer impact' label
>       - I have added a description and/or a video demo of the changes
>       below (e.g. a "before and after video")
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/github/docs/pull/7105#issuecomment-855325342>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AULWBJMO7IIFBKOYUQJY623TRLK4PANCNFSM46FDYTFQ>
> .
>
